### ISSUE TITLE
Create a One Sheet: Heart

### ISSUE THREAD

**Issue by henlatourrette** [hackforla/product-management#53]
### Overview
Create the project one sheet so it can be shared with stakeholders and funders.

### Action Items
- [x] Check to confirm project does not already have a one sheet
- [x] Speak to the team to get an overview of the project
- [x] Include project logo
- [x] Create history/last 6 months
- [x] Create roadmap/next 6 months
- [x] Include last updated date

### Resources/Instructions
Check issue #3 for sample one sheets
[Heart one sheet](https://docs.google.com/document/d/1Wj3AhwNMZBZfIyX-MFqCIVH1qD2QJR2e0FXYzUUL7bM/edit)

**Comment by henlatourrette** [comment]
Need to reach out to @ExperimentsInHonesty  on Slack - to get up to speed on the project

**Comment by ExperimentsInHonesty** [comment]
@henlatourrette Here is some stuff for you to review before we talk:
- An issue about finishing an article about heart: https://github.com/hackforla/marketing/issues/6
- The heart project page: https://www.hackforla.org/projects/heart

**Comment by henlatourrette** [comment]
Got access to the One Sheet Template now. I'm going to work on it for a 1st Draft with the information I have so far.

**Comment by henlatourrette** [comment]
Progress: Created a 1st Draft with the informtion gathered from the "article about heart". @ExperimentsInHonesty , would you take a look at my [draft ](https://docs.google.com/document/d/1Wj3AhwNMZBZfIyX-MFqCIVH1qD2QJR2e0FXYzUUL7bM/edit) and tell me what you think? :
Blocks: need to decide about "the roadmap" or leave it with just "results" - it looks like this project is already completed.
Availability: 5 hours or more.
ETA: By this week.

**Comment by ExperimentsInHonesty** [comment]
@henlatourrette Sorry for the delay.  
I reviewed the draft.
Moved one of the images
removed the roadmap
added a note for you to insert the logo

The new task items on this are
- [ ] add the partner logo
- [ ] revise the results language to be less !!.  The facts are themselves impressive. the sentences should be more matter of fact.

**Comment by henlatourrette** [comment]
Progress: I changed the language on the "results" paragraph and used bullet points instead. Added a section called "next steps" saying that the article will be published. Added the Github url as well.
Blocks: Looking for a good quality logo for the Partner.
Availability: Around 4 hours.
ETA: Tomorrow.

**Comment by henlatourrette** [comment]
Progress: I added the Partner logo and I think it's now finished already. 
Blocks: I believe I just need confirmation - or some final review and feedback from @ExperimentsInHonesty to consider it ready. Bonnie, would you [give it a look](https://docs.google.com/document/d/1Wj3AhwNMZBZfIyX-MFqCIVH1qD2QJR2e0FXYzUUL7bM/edit)?
Availability: 2 hrs.
ETA: Done, just waiting for some final feedback.

**Comment by ExperimentsInHonesty** [comment]
@henlatourrette I have made some edits.

**Comment by Olivia-Chiong** [comment]
@henlatourrette 
Progress:
Blocks:
Availability:
ETA:

**Comment by henlatourrette** [comment]
Hi team, I'm out for vacations this week - with little access to my computer. I will be back by next Monday Feb 8th.

**Comment by Olivia-Chiong** [comment]
hi @henlatourrette welcome back. Have you had the time work n this at all or will you need more time? Please update

Progress:
Blocks:
Availability:
ETA:

**Comment by henlatourrette** [comment]
Progress: Approved the edits and updated the working from present tense to past tense as suggested. I believe it's quite finished by now. @ExperimentsInHonesty would you give it a final look? 
Blocks: n/a
Availability: 4 hours - if required.
ETA: This week.

**Comment by ExperimentsInHonesty** [comment]
@henlatourrette welcome back from your trip!

I left you some comments and suggested changes on the one sheet.

**Comment by Olivia-Chiong** [comment]
@henlatourrette can you please provide an update on this?
Progress:
Blocks:
Availability:
ETA:

**Comment by henlatourrette** [comment]
Progress: Still working on the latest suggestion made by Bonnie.
Blocks: none yet.
Availability: 2 hours.
ETA: By Monday next week.

**Comment by henlatourrette** [comment]
**Progress:**
Made some changes in the "Results" part:
Added a new Bullet point and combined the previous two related to the numbers into just one.

**Blocks:** Bonnie told me to ask her if I didn't know what was wrong. I made some changes but I'm not sure whether those are the ones it needed. @ExperimentsInHonesty can you please [take a look at it ](https://docs.google.com/document/d/1Wj3AhwNMZBZfIyX-MFqCIVH1qD2QJR2e0FXYzUUL7bM/edit)again now?

**Availability:** 6 hours.
**ETA:** By this week.

**Comment by ExperimentsInHonesty** [comment]
Hi @henlatourrette I made a bunch of edits.  Can you review it now and tell me if you think it's good to go.

Thanks!  

Bonnie

**Comment by henlatourrette** [comment]
@ExperimentsInHonesty I just corrected some punctuation marks and I think it's now good to go. We can move forward with the next steps for the One Sheet.

**Comment by ExperimentsInHonesty** [comment]
@henlatourrette Thank you for your work on this!  I'll unassign you now.

**Comment by ExperimentsInHonesty** [comment]
This is ready to make into a pdf and update the chart  #3

**Comment by erigilg** [comment]
Hi @henlatourrette @ExperimentsInHonesty. I'm working on the slide of Heart for the ppt "[What is Hack for LA?](https://docs.google.com/presentation/d/16ayq8ur_bLy-3bYtXI3lyuQ2zvOcGUlszIaHZG6AK3M/edit#slide=id.gbfc3cf2292_0_119)". 
Could you make sure the [one sheet doc](https://docs.google.com/document/d/1Wj3AhwNMZBZfIyX-MFqCIVH1qD2QJR2e0FXYzUUL7bM/edit?usp=sharing) is in the right folder? I don't have access to it even though I'm already in the repo.

**Comment by henlatourrette** [comment]
> Hi @henlatourrette @ExperimentsInHonesty. I'm working on the slide of Heart for the ppt "[What is Hack for LA?](https://docs.google.com/presentation/d/16ayq8ur_bLy-3bYtXI3lyuQ2zvOcGUlszIaHZG6AK3M/edit#slide=id.gbfc3cf2292_0_119)".
> Could you make sure the [one sheet doc](https://docs.google.com/document/d/1Wj3AhwNMZBZfIyX-MFqCIVH1qD2QJR2e0FXYzUUL7bM/edit?usp=sharing) is in the right folder? I don't have access to it even though I'm already in the repo.

Hi Erika, send me your email address on Slack @ Henry Latourrette and I'll "share" the document to you from the Drive.

**Comment by ExperimentsInHonesty** [comment]
@Olivia-Chiong please sort out craziness above !! 🙈

**Comment by Olivia-Chiong** [comment]
@henlatourrette do not share the document directly. please make sure the document is moved to the correct One Sheet folder in the product management Google Drive so @erigilg can access it.

**Comment by henlatourrette** [comment]
> @henlatourrette do not share the document directly. please make sure the document is moved to the correct One Sheet folder in the product management Google Drive so @erigilg can access it.

Hi @Olivia-Chiong , actually I don't have access to its original folder so I cannot move the file. What I can do is to create a copy and put it into the Product Management Folder but I think it would not be the best option to have 2 different files. I see that @ExperimentsInHonesty is the owner. Anyway, I'll send a request on the Slack channel for admin stuff.

![image](https://user-images.githubusercontent.com/21250505/110220272-393cff80-7ea3-11eb-8aff-faceb499aae6.png)

**Comment by henlatourrette** [comment]
The Heart One Sheet file was moved to the correct folder [here](https://drive.google.com/drive/u/0/folders/1jEF6AwIHi5-82pdx-JsHoBuvxczw5Zyy).

**Comment by henlatourrette** [comment]
I can see that the PDF is now available [here ](https://github.com/hackforla/product-management/blob/master/project-one-sheets/Heart-Project-One-Sheet.pdf)and that the [Product One Sheet tracker](https://github.com/hackforla/product-management/issues/3) was updated. Is it OK to close this issue now @ExperimentsInHonesty , @Olivia-Chiong ? or should we keep it opened for future updates?

**Comment by henlatourrette** [comment]
I'm closing this issue as the updated PDF is already done.
